kikuchipy/signals/ebsd_master_pattern.py

```diff
@@ -289,3 +407,224 @@ class LazyEBSDMasterPattern(EBSDMasterPattern, LazySignal2D):
 
     def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
+
+
+def _get_direction_cosines(detector: EBSDDetector) -> Vector3d:
+    """
+    Get the direction cosines between the detector and sample as done in EMsoft
+    and [Callahan2013]_.
+
+    Parameters
+    ----------
+    detector : EBSDDetector
+        EBSDDetector object with a certain detector geometry.
+
+    Returns
+    ----------
+    r_g.unit : Vector3d
+        Vector3d object containing the direction cosines for each detector
+        pixel.
+    """
+
+    pc = detector.pc_emsoft()
```

Comment:
Per the docstring comment for the detector in `get_patterns()` above, are multiple PCs allowed here? If not, we should add a check.